service/src/intTest/resources/application.yml

```diff
@@ -40,7 +40,7 @@ gp2gp:
     ttl: ${GP2GP_MONGO_TTL:P7D}
     cosmosDbEnabled: ${GP2GP_COSMOS_DB_ENABLED:false}
   gpc:
-    url: ${GP2GP_GPC_GET_URL:https://orange.testlab.nhs.uk/B82617/STU3/1/gpconnect}
+    url: ${GP2GP_GPC_GET_URL:https://orange.testlab.nhs.uk/{ODS_CODE}/STU3/1/gpconnect}
```

Comment:
This is very similar to the ${ENV} placeholder. Should we do something completely different to avoid any confusion? Also update the README to document how to use this placeholder